golden_response/staging/get_places_in/state_county.json

```diff
@@ -1,135 +1,538 @@
+    "place": "geoId/06023"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06025"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06027"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06029"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06031"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06033"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06035"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06037"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06039"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06041"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06043"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06045"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06047"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06049"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06051"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06053"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06055"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06057"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06059"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06061"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06063"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06065"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06067"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06069"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06071"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06073"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06075"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06077"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06079"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06081"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06083"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06085"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06087"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06089"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06091"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06093"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06095"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06097"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06099"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06101"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06103"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06105"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06107"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06109"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06111"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06113"
+  },
+  {
+    "dcid": "geoId/06",
+    "place": "geoId/06115"
+  },
+  {
```

Comment:
@pradh this new addition seems wrong?

Reply:
This is a county so not totally off, but we should fix it next.